README.md

```diff
@@ -2,6 +2,8 @@
 
 # Fairness in Machine Learning - Mitigation algorithms
 
+[![Azure Notebooks](https://notebooks.azure.com/launch.png)](https://notebooks.azure.com/import/gh/microsoft/fairlearn)
```

Comment:
Actually this makes me wonder... This probably opens up notebooks with a view of all notebooks in the repo?

That's obviously awesome, but I'm also wondering whether we should add buttons for each of the algorithms (since they'll probably all get their own notebooks over time). So that's unrelated to your PR, but more of a question.

Reply:
maybe just in the table below?

Reply:
Unless Azure Notebook has solved the problem, otherwise widgets wouldn't work in Azure Notebooks. The widget has to be pre-installed by Azure Notebook on the image.

Reply:
I was going to wait and see what happens with the widget before putting an azure nb button in.